content/working_with/console/pages/_index.md

```diff
@@ -21,28 +21,26 @@ Below are the default set of pages (templates) presented per {{< param product_n
 
 ***main-sys_admin*** template applies to `sys-admin` users in all the tenants, and includes the following pages:
 
-* [Dashboard]({{< relref "working_with/console/pages/dashboard-page.md" >}}) - An overview of the existing executions and system statistics.
```

Comment:
It's hard to maintain and similar descriptions are on web-pages describing each page content.